python/mxnet/gluon/data/vision/transforms.py

```diff
@@ -581,3 +581,28 @@ def hybrid_forward(self, F, x):
         if is_np_array():
             F = F.npx
         return F.image.random_lighting(x, self._alpha)
+
+class RandomApply(Sequential):
+    """Apply a list of transformations randomly given probability
+
+    Parameters
+    ----------
+    Inputs:
+        - **transforms**: list of transformations
+        - **p**: probability
+
+    Outputs:
+        Transformed image.
+    """
+
+    def __init__(self, transform, p=0.5):
```

Comment:
Should the variable be transforms instead?

Reply:
Done.